docs/howto/custom-template-tags.txt

```diff
@@ -804,6 +804,20 @@ Notes:
   ``django.template.TemplateSyntaxError`` if they receive the wrong number or
   type of arguments.
 
+* You can implement a ``stream()`` method if your renderer needs to iterate.
```

Comment:
Maybe "needs to do iteration" or "do a loop" would be clearer. The idea is that a stream method can be implemented if the renderer does not render the result at once.